build-logic/src/main/kotlin/publishing/configurePom.kt

```diff
@@ -85,7 +86,14 @@ internal fun configurePom(project: Project, mavenPublication: MavenPublication,
             licenses {
               license {
                 name.set("Apache-2.0") // SPDX identifier
-                url.set(asfProject.licenseUrl)
+                url.set(gitInfo.rawGithubLink("LICENSE"))
+                comments.set(
+                  """
+                  NOTICE: ${gitInfo.rawGithubLink("NOTICE")}
+                  DISCLAIMER: ${gitInfo.rawGithubLink("DISCLAIMER")}
```

Comment:
Adding these links for release builds sounds valuable to me, but given current impl. of `GitInfo` for non-release builds (using `HEAD`) I do not think the links would be useful or even correct.

Could we identity whether `GetInfo` has a solid tag and avoid adding links if it does not?